src/Uno.Toolkit.UI/Controls/TabBar/TabBar.xaml

```diff
@@ -75,28 +75,58 @@
 							<VisualStateGroup x:Name="OrientationStates">
 								<VisualState x:Name="Horizontal">
 									<VisualState.Setters>
-										<Setter Target="SelectionIndicatorPresenter.VerticalContentAlignment" Value="Stretch" />
-										<Setter Target="SelectionIndicatorPresenter.HorizontalContentAlignment" Value="Left" />
+										<Setter Target="BelowSelectionIndicatorPresenter.VerticalContentAlignment" Value="Stretch" />
+										<Setter Target="BelowSelectionIndicatorPresenter.HorizontalContentAlignment" Value="Left" />
+										<Setter Target="AboveSelectionIndicatorPresenter.VerticalContentAlignment" Value="Stretch" />
+										<Setter Target="AboveSelectionIndicatorPresenter.HorizontalContentAlignment" Value="Left" />
 									</VisualState.Setters>
 								</VisualState>
 								<VisualState x:Name="Vertical">
 									<VisualState.Setters>
-										<Setter Target="SelectionIndicatorPresenter.VerticalContentAlignment" Value="Top" />
-										<Setter Target="SelectionIndicatorPresenter.HorizontalContentAlignment" Value="Stretch" />
+										<Setter Target="BelowSelectionIndicatorPresenter.VerticalContentAlignment" Value="Top" />
+										<Setter Target="BelowSelectionIndicatorPresenter.HorizontalContentAlignment" Value="Stretch" />
+										<Setter Target="AboveSelectionIndicatorPresenter.VerticalContentAlignment" Value="Top" />
+										<Setter Target="AboveSelectionIndicatorPresenter.HorizontalContentAlignment" Value="Stretch" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+							<VisualStateGroup x:Name="IndicatorPlacementStates">
+								<VisualState x:Name="Above">
```

Comment:
given Above is the dp default value, you can have these setters as the default states,
and leave this empty